main.cpp

```diff
@@ -107,10 +107,50 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
-
-
-
-
+struct Person
+{ 
+    int age; 
+    int height; 
+    float hairLenght; 
+    float GPA; 
+    unsigned int SATScore; 
+    float distanceTraveled; 
+
+    struct Foot 
+    { 
+        int numToes; 
+        float footLengthInCM;
+        float normalStepLengthInCM;
+        float speedAdjustedSteplength;
+
+        float stepSize(float stepSpeedMultiplicationFactor)
+        {
+            speedAdjustedSteplength = normalStepLengthInCM * stepSpeedMultiplicationFactor;
+
+            return speedAdjustedSteplength;
+        }
+    }; 
```

Comment:
This is missing a `stepForward()` function.
Re-watch the video and copy the `run()` code exactly. 
Then revise the `Foot` class to make the code compile.
```
 1) write the definition for the Type that leftFoot and rightFoot are instantiations of.
    don't forget to define and implement the member functions 'stepForward()' and 'stepSize()'
    you should be able to deduce the return type of those functions based on their usage in Person::run()
    You'll need to insert the Person struct from the video in the space below.
```
- [ ] revise